packages/fiori/test/pages/Search.html

```diff
@@ -18,8 +18,16 @@
 
 	<ui5-search>
```

Comment:
Perhaps add a separate test sample pointing that it shows byline items, instead of polluting the old one's, it will be easier for testing and testers.